### DISCUSSION TITLE
Github Documentation

### DISCUSSION THREAD

**Discussion by kddKena** [community/community#33498] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi guys, I wanna create a website like a Github Documentation (docs.github.com), please help me brother and sisters. I need your help, thanks!!!

**Comment by airtower-luna** [comment, marked as answer]
The sources for `docs.github.com` are here: https://github.com/github/docs

There is a [development guide](https://github.com/github/docs/blob/main/contributing/development.md), that's probably a good place to start adapting it for whatever you need. Make sure to follow the licenses, and good luck! :slightly_smiling_face:

**Reply by kddKena** [reply]
Thanks my bro or sis 😘😘
